src/udgl/menu/MenuList.svelte

```diff
@@ -58,7 +58,6 @@ ul {
   margin:0;
   padding:0;
   background-color: white;
-  border: 1px solid var(--line-gray-01);
```

Comment:
Are there no other `ul`s which will be affected by this?

Reply:
Really good catch! Though oddly enough I didn't find visual inconsistencies in the top-right menus for example (which use this) but still this needs to be reverted.